package.json

```diff
@@ -13,31 +13,31 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "babel-core": "6.16.0",
+    "babel-core": "6.23.1",
     "babel-eslint": "7.1.1",
-    "babel-loader": "6.2.7",
-    "babel-plugin-transform-class-properties": "6.22.0",
-    "babel-plugin-transform-runtime": "6.15.0",
-    "babel-preset-es2015": "6.18.0",
-    "babel-preset-react": "6.22.0",
+    "babel-loader": "6.3.2",
+    "babel-plugin-transform-class-properties": "6.23.0",
+    "babel-plugin-transform-runtime": "6.23.0",
+    "babel-preset-es2015": "6.22.0",
+    "babel-preset-react": "6.23.0",
     "babel-preset-react-hmre": "1.1.1",
     "cross-env": "3.1.4",
-    "css-loader": "0.14.5",
-    "eslint": "3.14.0",
-    "eslint-config-airbnb": "14.0.0",
+    "css-loader": "0.26.2",
```

Comment:
this is intentionally 0.14.5 because it's way faster to build and re-build. See https://www.slideshare.net/grgur/webpack-react-performance-in-16-steps